ui/src/app/ErrorInfo/ErrorInfo.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Popover, Icon, Tooltip } from '@patternfly/react-core';
+import ExclamationCircleIcon from '@patternfly/react-icons/dist/esm/icons/exclamation-circle-icon';
+import { DeploymentStepStatusData } from '@app/apis/types';
+
+interface IErrorInfoProps {
+  stepStatusData: DeploymentStepStatusData;
+}
+
+export const ErrorInfoPopover = ({ stepStatusData }: IErrorInfoProps) => {
+  const borders = {
+    border: '1px solid rgb(0,0,0)',
+  };
+
+  let data = (
```

Comment:
This should be `const`. 
Also, how about doing this with patternfly's table?
https://www.patternfly.org/v4/components/table#composable-basic
Or we can do it in another PR.